buildSrc/src/main/kotlin/com/datadog/gradle/Dependencies.kt

```diff
@@ -58,7 +62,13 @@ object Dependencies {
 
     object Libraries {
 
-        const val TracingOt = "com.datadoghq:dd-trace-ot:${Versions.TracingOt}"
+        @JvmField
+        val TracingOt = arrayOf(
+            "io.opentracing:opentracing-api:${Versions.OpenTracing}",
+            "io.opentracing:opentracing-noop:${Versions.OpenTracing}",
+            "io.opentracing:opentracing-util:${Versions.OpenTracing}",
+            "org.slf4j:slf4j-api:${Versions.SLF4J}"
```

Comment:
But why do we still need this if we strip all the lines when we clone the code ?

Reply:
Good call ;)